libyara/modules/pe.c

```diff
@@ -438,22 +431,9 @@ int64_t pe_rva_to_offset(
     }
   }
 
-  // Everything before the first section seems to get mapped straight
-  // relative to ImageBase.
-
-  if (rva < lowest_section_rva)
-  {
-    section_rva = 0;
-    section_offset = 0;
-    section_raw_size = pe->data_size;
-  }
-
-  // Many sections, have a raw (on disk) size smaller than their in-memory size.
-  // Check for rva's that map to this sparse space, and therefore have no valid
-  // associated file offset.
-
-  if ((rva - section_rva) >= section_raw_size)
-    return -1;
+  // Apparently, the loader maps data that resides between sections in
```

Comment:
The returned offset is going to be equal to the passed RVA for any RVA not contained within a section. What if the value passed to pe_rva_to_offset is an offset within the PE overlay? The value is going to be accepted as a valid RVA and is going to be returned as the corresponding offset. But I think the PE overlay is not mapped into memory by the loader? I don't have a Windows machine around to verify it. Can you confirm if that's the intended behavior?